docs/helpers/Nightmare.md

```diff
@@ -399,43 +440,53 @@ I.fillField({css: 'form#login input[name=username]'}, 'John');
 
 #### Parameters
 
--   `field` **([string][3] \| [object][4])** located by label|name|CSS|XPath|strict locator.
--   `value` **[string][3]** text value to fill.
+-   `field` ([string][3] \| [object][4]) located by label|name|CSS|XPath|strict locator.
+-   `value` [string][3] text value to fill.
+    
+
 
 ### grabAttributeFrom
 
 Retrieves an attribute from an element located by CSS or XPath and returns it to test.
 An array as a result will be returned if there are more than one matched element.
-Resumes test execution, so **should be used inside async with `await`** operator.
+Resumes test execution, so should be used inside async with `await` operator.
 
 ```js
 let hint = await I.grabAttributeFrom('#tooltip', 'title');
 ```
 
 #### Parameters
 
--   `locator` **([string][3] \| [object][4])** element located by CSS|XPath|strict locator.
--   `attr` **[string][3]** attribute name.
+-   `locator` ([string][3] \| [object][4]) element located by CSS|XPath|strict locator.
+-   `attr` [string][3] attribute name.
+
+Returns [Promise][8]&lt;[string][3]> attribute value
 
-Returns **[Promise][8]&lt;[string][3]>** attribute value
 
-### grabCookie
 
-{{> grabCookie}}
+### grabCookie
 
-Returns cookie in JSON format. If name not passed returns all cookies for this domain.
+Gets a cookie object by name.
+If none provided gets all cookies.
 
-Multiple cookies can be received by passing query object:
+-   Resumes test execution, so should be used inside async with `await` operator.
 
 ```js
-I.grabCookie({ secure: true});
+let cookie = await I.grabCookie('auth');
+assert(cookie.value, '123456');
 ```
 
-If you'd like get all cookies for all urls, use: `.grabCookie({ url: null }).`
-
 #### Parameters
 
--   `name`  
+-   `name`  cookie name. 
+
+Returns [Promise][8]&lt;[string][3]> attribute value
+
+
+
+Returns any Promise<object> Cookie in JSON format. If name not passed returns all cookies for this domain.Multiple cookies can be received by passing query object:```js
+I.grabCookie({ secure: true});
```

Comment:
Either multiple string or without `js`.

`js` is prepend string:
It looks like:```js I.grabCookie({ secure: true}); ```asdasd 

When multiple string:
```js
I.grabCookie({ secure: true});
```